build.gradle

```diff
@@ -140,7 +140,7 @@ publishing {
     mavenJava(MavenPublication) {
       groupId = 'com.datastax'
       artifactId = 'cass-config-builder'
-      version = "1.0.0-" + getDate()
```

Comment:
what was this about?

Reply:
The mavenJava logic builds a jar and publishes it to a maven repository.